tests/Security/TwoFactor/Trusted/TrustedCookieResponseListenerTest.php

```diff
@@ -200,6 +191,59 @@ public function provideRequestHostName(): array
             ['2001:0db8:85a3:0000:0000:8a2e:0370:7334'],
         ];
     }
+
+    /**
+     * @test
+     */
+    public function onKernelResponse_cookieSecureAuto_secureRequest(): void
```

Comment:
Can we rename please? `onKernelResponse_cookieSecureAutoOnSecureRequest_setSecureCookie`